### PR TITLE
Resource check

### PR DESCRIPTION
**Pull request by rbrady**

### CODE REVIEW

**Review by jistr — CHANGES_REQUESTED** [review]
Does this need a rebase on top of the latest main branch? The diff looks like the PR is introducing export/import for projects, but that was already merged earlier. So there's a lot of irrelevant code in the review diff.

**Comment by rbrady** [comment]
> Does this need a rebase on top of the latest main branch?

Rebased.

File: os_migrate/plugins/module_utils/serialization.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def new_resources_file_struct():
     return data
 
 
-def add_or_replace_resource(resources, resource_or_data):
+def add_or_replace_resource_old(resources, resource_or_data):
```
Comment (jistr) [inline os_migrate/plugins/module_utils/serialization.py:19]: We can delete this `_old` function now, right? The refactored function looks good, and the rest of the patch too. :+1:

**Review by jistr — CHANGES_REQUESTED** [review]

**Review by jistr — APPROVED** [review]
Thanks!

**Pull request merged**